packages/manager/modules/storages/src/netapp/netapp.module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import component from './netapp.component';
+import share from './share';
+
+import routing from './netapp.routing';
+import service from './netapp.service';
+
+const moduleName = 'ovhManagerNetapp';
```

Comment:
Please keep module name naming convention.

```suggestion
const moduleName = 'ovhManagerStorageNetapp';
```